### PR TITLE
Update defibrillator preset

### PR DESCRIPTION
**Pull request by nlehuby**

### CODE REVIEW

File: data/presets/fields/defibrillator_location.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (quincylvania) [inline data/presets/fields/defibrillator_location.json]: We usually use subdirectories for tags with colons. Could you move/rename this file to `fields/defibrillator/location.json`?

File: data/presets/fields/defibrillator_location.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    "key": "defibrillator:location",
+    "type": "textarea",
+    "label": "Precise location of the defibrillator"
```
Comment (quincylvania) [inline data/presets/fields/defibrillator_location.json]: Can we change the name to something shorter like "Location Description"?

**Review by quincylvania — CHANGES_REQUESTED** [review]
Looks fine, just a couple details I noticed.

**Comment by nlehuby** [comment]
Thanks for your review.

**Review by quincylvania — APPROVED** [review]

**Comment by quincylvania** [comment]
@nlehuby Thanks for your contribution!

**Pull request merged**